tp016/README.md

```diff
@@ -0,0 +1,167 @@
+# TP 16 Creation vs Writing
+
+```yaml
+TP: 16
+Title: Creation vs Writing
+Author(s): Daniel Buchner (@csuwildcat)
+Comments URI: https://github.com/TBD54566975/technical-proposals/discussions/1
+Status: Draft
+Created: March 13, 2023
+Updated: March 13, 2023
+```
+
+## Problem Statement
+
+It can be cumbersome/strange in dev UX terms to have to replay immutable properties of a record for subsequent Writes. The optimal setup would be to have immutable properties cast once, upon creation of a new record, and only be required to provide changes to mutable values you wish to modify in subsequent Writes.
+
+## Considerations
+
+For this propsal, create the following distinction in your mind regarding the properties that make up a record entry: 
+
+Immutable props:
+
+- Interface
+- Method
+- Protocol
+- Schema
+- dataFormat
+- dateCreated
+- parentCid
```

Comment:
Nit: In `dwn-sdk-js` and in the spec, this is `parentId` instead of `parentCid`. I honestly prefer `parentCid` since it _is_ a CID. Either is fine by me, but want to make sure we're consistent.

Regardless of naming, why is this field needed and why is it immutable? Upon record creation (regardless of which message type we choose), we would have no 'parent' to take the ID of.

Reply:
is `parentId` a cid? i thought `parentId` was the `recordId` of the parent

Reply:
It's probably not a CID, just a typo. (record ID makes more sense here, but there is a CID for Commit ancestor pointers)